packages/signers/pjs-signer/src/pjs-signed-extensions-mappers.ts

```diff
@@ -28,13 +32,14 @@ export const CheckGenesis = ({
 export const CheckNonce = ({
   value,
 }: SignedExtension): { nonce: HexString } => {
-  return { nonce: toPjsHex(compact.dec(value)) }
+  // nonce is a u32 in pjs => 4 bytes
```

Comment:
only god knows why, because when encoded in the sign-extension is a compact... 🤷

Thanks for adding the comment!